core/src/com/weepingwasp/eventManager/EventManager.kt

```diff
@@ -16,10 +16,14 @@ fun pushEvent(event: Event) {
 enum class EventType() {
     PLAYCARD,
     ENDTURN,
+    STARTTURN,
```

Comment:
The coding convention for Kotlin suggests uppercase names separated by an underscore, so STARTTURN should be START_TURN